app.js

```diff
@@ -9,6 +17,12 @@ app.get("/api", getApi);
 
 app.get("/api/topics", getTopics);
 
+app.get("/api/articles/:article_id", getArticle);
+
 app.all("*", badUrlErrorHandler);
 
+app.use(customErrorHandler);
+
+app.use(postgressErrorHandler);
```

Comment:
To complete our error handling middleware, what should the app send if neither of these error handlers handle the error?